test/hotspot/jtreg/compiler/c2/Test8202414.java

```diff
@@ -34,10 +36,18 @@
 import java.lang.reflect.Field;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
+import jtreg.SkippedException;
 
 public class Test8202414 {
 
     public static void main(String[] args) {
+        // Some CPUs (for example, ARM) does not support unaligned
+        // memory accesses. This test may cause JVM crash due to
+        // alignment check failure on such CPUs.
+        if (!jdk.internal.misc.Unsafe.getUnsafe().unalignedAccess()) {
+          throw new SkippedException(
```

Comment:
nit: I don't think we need a line break here.

Reply:
Previously, all lines within this file were no longer than 80 chars, so I decided to follow the same restriction.